### ISSUE TITLE
Description of the repo is missing

### ISSUE THREAD

**Issue by budhrg** [minishift/minishift#162]
A nice description for repo would be good to give brief introduction of the minishift to the users.

**Comment by thatdocslady** [comment]
We're doing a lot of doc restructuring now (see issue #138 for example), so I'll handle this after the major changes are merged.

@budhrg in the meantime could you help write more info here so that I can transport it more easily into the readme once we're ready?

**Comment by budhrg** [comment]
@thatdocslady We can discuss here to get the best description for our repo.

WDYT ? @jimmidyson @hferentschik @LalatenduMohanty @praveenkumar

**Comment by hferentschik** [comment]
@thatdocslady I think what @budhrg means is the description in the GitHub Settings. You see it at the root of the repo - https://github.com/minishift/minishift

The README already has a "What is Minishift?"

> Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs a single-node OpenShift cluster inside a VM on your laptop for users looking to try out OpenShift or develop with it day-to-day.

I pretty much like the second half of this. Should be good enough, but we can discuss better, more concise wording.

**Comment by budhrg** [comment]
> I think what @budhrg means is the description in the GitHub Settings

Ya the Github settings. 
Repo without description is always felt incomplete in my world :smile: .

**Comment by budhrg** [comment]
> Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs a single-node OpenShift cluster inside a VM on your laptop for users looking to try out OpenShift or develop with it day-to-day.

I don't think this long description will help. It looks exactly like what is present in README and for me it sounds duplicate.

It should be short and catchy. Check the minikube description https://github.com/kubernetes/minikube.

**Comment by hferentschik** [comment]
> I don't think this long description will help. It looks exactly like what is present in README and for me it sounds duplicate.

Fair enough. You got your wish. It says "Run OpenShift locally" for now. Works for me. If someone has any better idea, let's discuss it here. Otherwise we can resolve this.

**Comment by jimmidyson** [comment]
Done.

**Comment by budhrg** [comment]
Cool :+1:

**Comment by LalatenduMohanty** [comment]
> Run OpenShift locally" 

+1
